python-src/resources.py

```diff
@@ -1,6 +1,8 @@
 import datetime
 import random
 
+from google.appengine.api.channel import channel
```

Comment:
I tried pip installing the `google-api-python-client` and I now have this in my lib folder.

<img width="305" alt="screen shot 2016-12-27 at 14 39 53" src="https://cloud.githubusercontent.com/assets/7635284/21500774/8877ec76-cc42-11e6-871c-83d187fb7d1b.png">

What am I missing here? (I can't import `google.appengine.api.channel`)